src/instructlab/model/evaluate.py

```diff
@@ -301,6 +304,15 @@ def get_backend(backend, model):
     return backend
 
 
+def dump_results_to_json(benchmark: str, results: dict):
+    """dump the given results dictionary from benchmark into a JSON file"""
+    timestamp = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+    filepath = os.path.join(DEFAULTS.EVAL_DATA_DIR, f"{benchmark}_{timestamp}.json")
```

Comment:
Should use the param rather than the default for EVAL_DATA_DIR

Reply:
What param?

Reply:
output_dir